pkg/measure-config/src/measurefs.go

```diff
@@ -0,0 +1,278 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"log"
```

Comment:
It probably makes sense to pick up the logrus setup with use (to get stack trace, etc).
Looking at pkg/newlog I think the minimal invocation in main() would be to call
	_, log = agentlog.Init("measure-config")

Reply:
it doesn't work. We are in onboot section and login facilities are not running. I may try using logrus directly

`agent measure-config[1]: spoofStdFDs: Error opening duplicate stdout with fd: 7
2022-11-08T18:45:35Z,onboot.005-measure-config.out;printToFile failed open /persist/log/reboot-reason.log: no such file or directory
2022-11-08T18:45:35Z,onboot.005-measure-config;{"file":"/measure-config/vendor/github.com/lf-edge/eve/pkg/pillar/agentlog/agentlog.go:585","func":"github.com/lf-edge/eve/pkg/pillar/agentlog.spoofStdFDs","level":"fatal","msg":"spoofStdFDs: Error opening duplicate stdout with fd: 7","pid":1,"source":"measure-config","time":"2022-11-08T18:45:35.174366462Z"}`

Reply:
Ah - agentlog makes assumptions about /persist.
Using logrus with the settings to add the stack trace and a name of the service makes sense. 
Should be able to extract how to set those up from agentlog.